Archipelago.HollowKnight/DeathLinkSupport.cs

```diff
@@ -0,0 +1,336 @@
+            "@ took damage equal to or more than their current HP.",
+            "@ made a fatal mistake.",
+            "@ threw some shade at @.",
+        };
+
+        public readonly static List<string> UnknownMessages = new()
+        {
+            "@ has died in a manner most unusual.",
+            "@ found a way to break the game, and the game broke @ back.",
+            "@ has lost The Game",
+        };
+
+        public readonly static Dictionary<int, List<string>> MessagesByType = new()
+        {
+            {
+                1,  // Deaths from enemy damage 
+                new List<string> {
+                    "@ has discovered that there are bugs in Hallownest.",
+                    "@ should have dodged.",
+                    "@ should have jumped.",
+                    "@ significantly mistimed their parry attempt.",
+                    "@ should have considered equipping Dreamshield.",
+                    "@ must have never fought that enemy before.",
+                    "@ did not make it to phase 2.",
+                }
+
+            },
+            {
+                2,  // Deaths from spikes
+                new List<string> {
+                    "@ was in the wrong place.",
+                    "@ mistimed their jump.",
+                    "@ didn't see the sharp things.",
+                    "@ didn't see that saw.",
+                    "@ fought the spikes and the spikes won.",
+                    "@ sought roses but found only thorns.",
+                    "@ was pricked to death.",
+                }
+            },
+            {
+                3,  // Deaths from acid
+                new List<string> {
+                    "@ was in the wrong place.",
+                    "@ mistimed their jump.",
+                    "@ forgot their floaties.",
+                    "What @ thought was H2O was H2SO4.",
+                    "@ wishes they could swim.",
+                    "@ used the wrong kind of dive.",
+                    "@ got into a fight with a pool of liquid and lost.",
+                }
+            },
+            {
+                999,  // Deaths in the dream realm
+                new List<string> {
+                    "@ dozed off for good.",
+                    "@ was caught sleeping on the job.",
+                    "@ sought dreams but found only nightmares.",
+                    "@ got lost in Limbo.",
+                    "Good night, @.",
+                    "@ is resting in pieces.",
+                    "@ exploded into a thousand pieces of essence.",
+                    "Hey, @, you're finally awake.",
+                }
+            },
+        };
+    };
+
+    public class DeathLinkSupport
+    {
+        public readonly static DeathLinkSupport Instance = new();
+        private DeathLinkService service = null;
+        public bool Enabled { get; private set; } = false;
+        private DeathLinkType Mode { get => Archipelago.Instance.SlotOptions.DeathLink; }
```

Comment:
```suggestion
        private DeathLinkType Mode => Archipelago.Instance.SlotOptions.DeathLink;
```

Lambda syntax getter only property